### PR TITLE
Explorer Context Menu Handlers

### PR DESCRIPTION
**Pull request by liamjm**
Reviewer: @destijl, @joachimmetz

[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/133)

### CODE REVIEW

File: definitions/windows.yaml [hunk 1]
```diff
@@ -131,6 +131,22 @@ sources:
 labels: [Logs]
 supported_os: [Windows]
 ---
+name: ExplorerContextMenuHandlers
+doc: Handlers for subcommands on context menu
+- type: REGISTRY_VALUE
```
Comment (joachimmetz) [inline definitions/windows.yaml]: add sources:

**Comment by joachimmetz** [comment]
Validator is failing, looks like "sources" is missing

File: definitions/windows.yaml (hunk 1 above)
Comment (liamjm) [inline definitions/windows.yaml]: done.
Thanks.

**Comment by joachimmetz** [comment]
LGTM

**Pull request merged**